### ISSUE TITLE
Use python logging module instead of print statements

### ISSUE THREAD

**Issue by sharkwouter** [sharkwouter/minigalaxy#548]
At the moment we have a lot of print statements and we have a debug mode. This could be done better with the logging module. It could be a global logger. The logs should be written to a file.

**Comment by orende** [comment]
This has been done as part of #573 except for this [line](https://github.com/sharkwouter/minigalaxy/blob/master/minigalaxy/launcher.py#L256) which I can't quite figure out if it's intentionally using a print statement or not.

**Comment by sharkwouter** [comment]
True, I'll go over our implementation one more time and see if we're at the point where it should be.
